src/main/java/life/qbic/subscriptions/SpringBootRestTemplateApplication.java

```diff
@@ -1,9 +1,19 @@
 package life.qbic.subscriptions;
 
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.ConfigurableApplicationContext;
 
+@OpenAPIDefinition(
+    info = @Info(
+        title = "Subscription Service",
+        version = "1.0.0",
+        description = "Cancel project update subscriptions and request cancel tokens.",
+        contact = @Contact(name = "Sven Fillinger", email = "sven.fillinger@qbic.uni-tuebingen.de")
```

Comment:
hier könnten wir auch zendesk verlinken

Reply:
```suggestion
        contact = @Contact(name = "Helpdesk", email = "support@qbic.zendesk.com")
```

Reply:
1.) I prefer the English language on Github.com :)
2.) Sure, however these is no consumer of the API outside of our team, so it will end up on my desk anyway